engine/src/main/java/com/datatorrent/stram/StramClient.java

```diff
@@ -207,22 +207,10 @@ public void stop()
     HashMap<String, String> sourceToJar = new HashMap<>();
 
     for (Class<?> jarClass : jarClasses) {
-      if (jarClass.getProtectionDomain().getCodeSource() == null) {
-        // system class
-        continue;
+      String jar = JarHelper.getJar(jarClass);
```

Comment:
Are there no other standard utilities or libraries that can get this information?